configure_venv.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
+setlocal
```

Comment:
@dnwillia-ansys Out of curiosity, why do we need to use `setlocal` here as we don't define any variable?

Reply:
Yeah, it's technically not needed here.  Force of habit from my side.

Reply:
That's fine by me. Thanks Dan.